### DISCUSSION TITLE
可以加一个倍速播放的功能不？

### DISCUSSION THREAD

**Discussion by fedono** [linuxdeepin/developer-center#3259] [Features Request & Ideas | 特性请求 & 头脑风暴]
本地的音频播放，希望深度能加一个倍速播放的功能，如果暂时加不了的话，有其他可用的播放器推荐的不？

**Comment by BLumia** [comment]
倍速播放的需求不清楚后续会不会引入 @meiyixiang 。

其它播放器的话，可以试试 VLC。

**Reply by fedono** [reply]
好嘞，感谢，我目前用深度影院来听，这个可以倍数。deepin 系统确实越来越好了 。
